podcasts/ThemeableSelectionView.swift

```diff
@@ -65,4 +68,13 @@ class ThemeableSelectionView: UIView {
         backgroundColor = AppTheme.colorForStyle(style, themeOverride: themeOverride)
         layer.borderColor = isSelected ? AppTheme.colorForStyle(selectedStyle, themeOverride: themeOverride).cgColor : AppTheme.colorForStyle(unselectedStyle, themeOverride: themeOverride).cgColor
     }
+
+    private func updateAccessibilityLabel() {
+        if unselectedAccessibilityLabel == nil {
+            unselectedAccessibilityLabel = accessibilityLabel
+        }
+        let unselectedAccessibilityLabel = self.unselectedAccessibilityLabel ?? ""
+        let selectedAccessibilityLabel = "\(L10n.statusSelected), \(unselectedAccessibilityLabel)"
+        accessibilityLabel = isSelected ? selectedAccessibilityLabel : unselectedAccessibilityLabel
```

Comment:
@emilylaguna  thank you so much for suggesting to use accessibilityValue. I did not add new string for unselected as that is not needed because it is enough to indicate when something is selected. In fact, this is how Apple's controls work. Try setting isSelected to true on UIButton and word selected will be pronounced when button is selected. However, when button is not selected it wont pronounce unselected but will just read its accessibility label.

Reply:
Wow, this is even better.